### DISCUSSION TITLE
NFT entries for Sorobanathon StellarQuest 5 series

### DISCUSSION THREAD

**Discussion by StellarGee** [stellar/sorobanathon#144] [General]
I am not an NFT artist at all, but try to support the Stellar NFT community as project member, NFT collector and by trying to spread the word of Stellar on socials. I wanted to give it my first try minting my own NFTs, because I love StellarQuest (a big thx to Tyler and the whole SQ team) and am super excited for what`s to come for Stellar and it`s community with Soroban coming to life.
I wanted to pay hommage to the pixel orientated style of StellarQuest and that`s my story of the StellarQuest5 in my NFTs:

Prologue - Through the sand
https://litemint.com/items/GB5Y7VHZRHCKEUKPVVMG4DASZOZMWJEYIS6BC4U2GNJV5EHMM5MLKGWZ/SQ5NFT1

Quest 1 - The temple entrance
https://litemint.com/items/GAX7HU72H5FBJ6TZABDQBQAJ3DJCODQLDBZ7CUZHRGLNV3FRMXO5FVN6/SQ5NFT2

Quest 2 - The temple hallways
https://litemint.com/items/GCQQMCMDNJKLSIREDEVQFRDQWANA7QNJ5T4XGFWPCN6RW4PJAYP3GWB3/SQ5NFT3

Quest 3 - Through the flames
https://litemint.com/items/GBIFP4XKDZPUHCLJT5LI5RDSYMLEZNXEQOSKYP4HOYHDDVX7DTJ5D4TA/SQ5NFT4

Quest 4 - The Lorekeeper
https://litemint.com/items/GDLO4OIUO3LPZY5E3OA7Z6Z3XVYFSANXLKAB45BLEI5SS6GS5F7MFUC5/SQ5NFT5

Quest 5 - The Golden Soroban
https://litemint.com/items/GAE5Z7VIBMFVNJGJZ66DQIV4FLYXXMEMIWMNZJQ646WKT4CJ77GPEGDU/SQ5NFT6

Quest 6 - The Vanishing
https://litemint.com/items/GCLG2GLHANT7XLZ4BFDZ4MZDVDK6CLWU6K6FNVOEUJN5I3K3FV7MMSWS/SQ5NFT7

**Comment by stellarjohn** [comment]
Hi [StellarGee](https://github.com/StellarGee)!

Congratulations! Your contribution to has been accepted! 🎉
Your submission may qualify for a reward of 5100 XLM! One last step: to receive that reward, please fill out this [form](https://stellarform.typeform.com/Sorobanathon) completely and accurately.

Please let us know if you have any questions.

Cheers!
John

**Comment by StellarGee** [comment]
Wow, this is amazing! If I will seriously receive money I`ll invest it straight away completely into Stellar art, because our artists need support to survive this crypto/NFT - winter and they earn it all the way!
